ssl/time.c

```diff
@@ -11,8 +11,13 @@
 #include <openssl/err.h>
 #include "internal/time.h"
 
+OSSL_TIME OSSL_TIME_INFINITY = { ~(uint64_t)0 };
+OSSL_TIME OSSL_TIME_IMMEDIATE = { 0 };
```

Comment:
Put these in a separate source file.  That'll get rid of the linking problem that this PR currently experiences

Reply:
They should be const

Reply:
Makes sense @levitte, thanks.  No idea why I didn't realise :disappointed: